.github/pull_request_template.md

```diff
@@ -8,22 +8,19 @@
 
 ## Related Issue
 
-```{=html}
-<!--- if this closes an issue make sure include e.g., "closes #4"
-or similar - or if just relates to an issue make sure to mention
-it like "#4" -->```
+`{=html} <!--- if this closes an issue make sure include e.g., "closes #4" or similar - or if just relates to an issue make sure to mention it like "#4" -->`
```

Comment:
Eh, I'm not sure what this looks like when adding a PR, I never used these templates.  If this note is visible as written here when opening a PR and not shown after submitting, what I suppose it would do, it is fine for me.

(Personally I favor the use of `fixes #x` and dislike `closes` or maintainers that close issues before the solution is merged to the main branch, certainly from the viewpoint of a package user that encounters a problem and finds a similar issue closed already.  But this is personal and out of question here.  ;-) )